cmd/codegen/codegen.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/jenkins-x/jx/cmd/codegen/app"
+	"os"
+)
+
+func main() {
```

Comment:
So codegen is its own binary now. There is still a dependency to methods under `pgk`, however minimal. TBH, I am tempted to break the link completely and have a true standalone tool. There is not much benefit in sharing here. On the contrary.